kivy/uix/behaviors/touchripple.py

```diff
@@ -0,0 +1,312 @@
+'''
+Touch Ripple
+============
+
+.. versionadded:: 1.9.2
+
+This module contains `mixin <https://en.wikipedia.org/wiki/Mixin>`_ classes
+to add a touch ripple visual effect known from `Google Material Design
+<https://en.wikipedia.org/wiki/Material_Design>_` to widgets.
+
+For an overview of behaviors, please refer to the :mod:`~kivy.uix.behaviors`
+documentation.
+
+The class :class:`~kivy.uix.behaviors.touchripple.TouchRippleBehavior` provides
+rendering the ripple animation.
+
+The class :class:`~kivy.uix.behaviors.touchripple.TouchRippleButtonBehavior`
+basically provides the same functionality as
+:class:`~kivy.uix.behaviors.button.ButtonBehavior` but rendering the ripple
+animation instead of default press/release visualization.
+'''
+from kivy.animation import Animation
+from kivy.clock import Clock
+from kivy.graphics import CanvasBase
+from kivy.graphics import Color
+from kivy.graphics import Ellipse
+from kivy.graphics import ScissorPush
+from kivy.graphics import ScissorPop
```

Comment:
That's a matter of taste I guess. Personally I prefer one import per line. If there is a consensus among kivy developers preferring multiple imports per line I can change it for sure.